src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs

```diff
@@ -77,7 +77,7 @@ public void SqlConnectionInvalidParameters()
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout= null;"));
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout=1 1;"));
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout=1a;"));
-            Assert.Throws<ArgumentException>(() => new SqlConnection("Integrated Security=truee"));
+            Assert.Throws<ArgumentException>(() => new SqlConnection("Integrated Security=true"));
```

Comment:
I think this was intentionally misspelled.. now it doesn't throw exception 😏

Reply:
ahh yes. silly mistake on my part. fixed